lib/rubocop/cop/rspec/empty_line_after_hook.rb

```diff
@@ -41,14 +41,9 @@ class EmptyLineAfterHook < Cop
 
```

Comment:
Wondering where the `s` from `%<hook>s` is, since in the spec I can't see it:
```ruby
        before { do_something }
        ^^^^^^^^^^^^^^^^^^^^^^^ Add an empty line after `before`.
```

Reply:
`String#%` uses arguments similar to `Kernel#sprintf`, and the `s` refers to the `string` type. See https://ruby-doc.org/core-2.6.6/Kernel.html#method-i-sprintf